inc/plugins/archiving.php

```diff
@@ -356,7 +356,12 @@ function archiving_isAllowedToArchive($thread)
 			$partners[] = $row['uid'];
 		}
 	} elseif ($db->table_exists('scenetracker')) {
-		$partners = explode(",", $query);
+		$partners_name = explode(",", $query);
+		
+		foreach($partners_name as $name) {
+			$partner = get_user_by_username($name);
+			$partners[] = $partner['uid'];
+		}
```

Comment:
```suggestion
		$partnerNames = explode(",", $query);
		
		foreach ($partnerNames as $partnerName) {
			$partner = get_user_by_username($partnerName);
			$partners[] = $partner['uid'];
		}
```

Upsi, ja die Formatierungen haben gepasst. hab noch ein Vorschlag zu der Variablenbenennung dagelassen. mags wenn es einheitlich ist. Was sagst du?